R/assocTestSingle.R

```diff
@@ -7,7 +7,8 @@ setMethod("assocTestSingle",
           function(gdsobj, null.model, test=c("Score", "Score.SPA", "BinomiRare", "CMP"),
                    recalc.pval.thresh=0.05, fast.score.SE=FALSE, GxE=NULL,
                    sparse=TRUE, imputed=FALSE, male.diploid=TRUE, genome.build=c("hg19", "hg38"), 
-                   verbose=TRUE) {
+                   verbose=TRUE, geno.test=c('additive','dominant','recessive')) {
+
```

Comment:
I'm not sure that i want to use the term "test" again for the parameter here. How about `geno.coding`?

Reply:
I like `geno.coding`

Reply:
I also like `geno.coding`.